core/controller/src/main/scala/whisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -284,7 +284,12 @@ class ShardingContainerPoolBalancer(config: WhiskConfig, controllerInstance: Con
     activations.getOrElseUpdate(
       msg.activationId, {
         val timeoutHandler = actorSystem.scheduler.scheduleOnce(timeout) {
-          processCompletion(Left(msg.activationId), msg.transid, forced = true, invoker = instance)
+          processCompletion(
+            Left(msg.activationId),
+            msg.transid,
+            forced = true,
+            invoker = instance,
+            isLogCollectionFinished = true)
```

Comment:
Good find on this one, @cbickel!

It looks like `isLogCollectionFinished` is set to `true` for forced active acks. Can you describe the intended behavior for forced active acks with the new log flag?